lms/djangoapps/course_api/serializers.py

```diff
@@ -10,6 +10,7 @@
 from rest_framework import serializers
 
 from openedx.core.djangoapps.models.course_details import CourseDetails
+from openedx.core.djangoapps.content.course_overviews.models import CourseOverview
```

Comment:
this is imported, but not used

Reply:
Yeah, sorry about that. I had to accept the pull request quickly so that we don't block anyone with our fix.

By the way, this shouldn't be a problem, because this will only affect quality tests. It was resolved in the upstream changes I believe though.
We can revert the commit though and apply the necessary changes if necessary though.